README.md

```diff
@@ -356,7 +356,8 @@ database is encountered that is not prefixed the primary metastore is used to re
       name: primary
       remote-meta-store-uris: thrift://primaryLocalMetastore:9083
     federated-meta-stores:
-      - name: waggle_prod_
+      - name: waggle_prod
+        prefix: waggle_prod
```

Comment:
this is not necessary, by default the prefix is made up from the `name`+`_`. So `waggle_prod` becomes `waggle_prod_`

Reply:
Yeah, so this is what I changed, I added the `prefix` as the documentation refers to the `prefix` not the `name` so it was confusing. So it sounds like you can use either but the documentation isn't very clear on that. We should probably decide which one we prefer and then go through this whole document and make sure its consistent. I thought we separated `prefix` and `name` so that the one would show up in log files etc (a readable `name`) and the other would be used for prefixing which is clearer IMHO, but I can't remember all the details now.